target/xilinx/xilinx_ips/common.mk

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 ETH Zurich and University of Bologna.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Cyril Koenig <cykoenig@iis.ee.ethz.ch>
+
+all: load-artifacts $(PROJECT).xpr save-artifacts
+
+# Build IP
+xlnx_%.xpr:
+	$(vivado_env) $(VIVADO) -mode batch -source tcl/run.tcl
+
+save-artifacts:
+
+load-artifacts:
+
+clean:
+	@rm -rf ip/*
+	@mkdir -p ip
+	@rm -rf ${PROJECT}.*
+	@rm -rf component.xml
+	@rm -rf vivado*.jou
+	@rm -rf vivado*.log
+	@rm -rf vivado*.str
+	@rm -rf xgui
+	@rm -rf .Xil
+	@rm -rf tmp
+	@rm -rf .generated*
+
+.PHONY: clean save-artifacts load-artifacts
+
+#
+# Artifacts management (IIS internal)
+#
+
+ifeq ($(XILINX_USE_ARTIFACTS),1)
+
+# Note: We do not use Memora as it is bound to Git versionning
+# and not standalone on files hash / environment variables
+ARTIFACTS_PATH=/usr/scratch2/wuerzburg/cykoenig/memora/carfield
```

Comment:
can we set this as env variable in the CI script so that we do not have to write down the machine's path explicitly?

Reply:
Sure